pkgs/applications/networking/cluster/kns/default.nix

```diff
@@ -0,0 +1,39 @@
+{
+  stdenvNoCC,
+  lib,
+  fetchFromGitHub,
+  fzf,
+  kubectl,
+}:
+stdenvNoCC.mkDerivation {
+  pname = "kns";
+  version = "unstable-2023-04-25";
+
+  src = fetchFromGitHub {
+    owner = "blendle";
+    repo = "kns";
+    rev = "86502949c31432bd95895cfb26d1c5893c533d5c";
+    sha256 = "sha256-8AR/fEKPAfiKCZrp/AyJo3Ic8dH7SfncYZSdQA2GywQ=";
```

Comment:
Prefer `hash = ...` to `sha256 = ...` Both currently work, but the former will make it easier if the algorithm ever needs to change.

Since there could be different hashing algorithms in use, this make sense.

Reply:
done